Assets/MixedRealityToolkit-SDK/Features/UX/Interactable/Materials/Background.mat

```diff
@@ -0,0 +1,178 @@
+%YAML 1.1
```

Comment:
Can we use one of the MRTK standard materials here?

Reply:
Do we have a set of button materials?